### DISCUSSION TITLE
Nice work! I've created a simple plugin with this. Any plans on more field types in the future?

### DISCUSSION THREAD

**Discussion by qriouslad** [jeffreyvr/wp-settings#3] [Q&A]
Hi Jeffrey,

Greetings from Indonesia! Thank you for creating this awesome package! 

I've just tried it to create a very simple plugin [Elegant Scroll Top](https://github.com/qriouslad/elegant-scroll-top) by reading the documentation and watching your [video tutorial](https://vanrossum.dev/24-demo-of-the-wordpress-settings-package-video).

Any plans on adding more ready-to-use field types in the future? I like how intuitive this library is and was hoping to see more of what [WPSF framework](https://wpsf.gitbook.io/docs/) provides also made available in your package... at least the basic ones to begin with, e.g. checkbox (single and multiple), radio box, media upload, etc.

Have a great weekend!

Cheers,
Bowo

**Comment by jeffreyvr** [comment]
That's cool @qriouslad! 

The checkbox is already available actually, I seem to have forgotten to mention that in the readme file. 

```php
$section->add_option('checkbox', [
    'name' => 'option_1',
    'label' => __('Option 1', 'textdomain')
]);
```

I'm actually also making a package for [meta boxes](https://github.com/jeffreyvr/wp-meta-box). There I already have media uploads, colors etc. so it won't be to difficult to add that here too.

Will have to find time to do so. I mostly add things when I need them in a project myself :)

**Reply by qriouslad** [reply]
Noted on checkbox already available @jeffreyvr ... and I understand the nature of open souce projects like this, so, no hurry. :-)

Good to know you're working on a meta box package too! Will revisit in time.

Keep up the great work!
